submissions/vovan-zt/Frogger-Game/app.js

```diff
@@ -0,0 +1,123 @@
+const gridSize = {
+    width: 101,
+    height: 81,
+}
+
+const fieldSize = {
+    width: gridSize.width * 5,
+    height: gridSize.height * 6,
+    top: 0,
+    left: 0,
+}
+
+const startPlayerPosition = {
+    x: gridSize.width * 2,
+    y: gridSize.height * 5,
+}
```

Comment:
By convention static/global constants are put ALL CAPS. This way, when we see a variable name ALL_CAPS in the code we understand it is a constant.

See examples in bot's checklist.